src/Airspace/AirspaceParser.cpp

```diff
@@ -63,6 +63,34 @@ static constexpr AirspaceClassStringCouple airspace_class_strings[] = {
   { "RMZ", RMZ },
   { "MATZ", MATZ },
   { "GSEC", WAVE },
+  { "UNCLASSIFIED", UNCLASSIFIED },
+  { "RESTRICTED", RESTRICTED },
+  { "TMA", TMA },
```

Comment:
DANGER is missing

Reply:
Was already defined ere:

https://github.com/XCSoar/XCSoar/blob/303cb6eb44d8265afd880828e78f75504cadcdee/src/Airspace/AirspaceParser.cpp#L50

Rename it to` { "DANGER", DANGER }` ?

Reply:
No, add. That would break classic openair files. https://pyopenair.readthedocs.io/en/latest/openair.html

Reply:
Added

`  { "DANGER", DANGER2 },
`